cors/src/main/java/io/helidon/cors/LogHelper.java

```diff
@@ -68,8 +68,21 @@ void setAndLog(BiConsumer<HeaderName, Object> consumer, String note) {
         }
     }
 
+
+    static <T> void logIsRequestTypeNormalNoOrigin(boolean silent, CorsRequestAdapter<T> requestAdapter) {
+        if (silent || !CorsSupportHelper.LOGGER.isLoggable(DECISION_LEVEL)) {
+            return;
+        }
+        if (CorsSupportHelper.LOGGER.isLoggable(DECISION_LEVEL)) {
```

Comment:
You don't need this check, right? You already made it?

Reply:
Correct. Fixed.